pyccel/codegen/printing/fcode.py

```diff
@@ -3220,8 +3280,6 @@ def fcode(expr, filename, assign_to=None, **settings):
         the expression is assigned. Can be a string, ``Symbol``,
         ``MatrixSymbol``, or ``Indexed`` type. This is helpful in case of
         line-wrapping, or for expressions that generate multi-line statements.
-    precision : integer, optional
-        The precision for numbers such as pi [default=15].
     user_functions : dict, optional
         A dictionary where keys are ``FunctionClass`` instances and values are
         their string representations. Alternatively, the dictionary value can
```

Comment:
I just noticed that the signature has `**settings` while here we discuss `user_functions`. Is this a dictionary contained in `settings`?

Reply:
Fixed #1791